doc/Language/packages.pod6

```diff
@@ -179,6 +179,14 @@ from a type name by use of the C<::> postfix:
 =for code :skip-test
 MyType::<$foo>
 
+=head2 Class member lookup
+
+Class attributes and methods are stored in the class meta object and can be looked up
```

Comment:
Isnt' 'lookup' searching only for methods (not members)?

Reply:
It'll show autogenerated accessor methods for public attributes. clarified that in https://github.com/perl6/doc/commit/4f53943fa08c1a1ef73587b23e8cbde5783f71e1